indexer/database/bridge_transfers.go

```diff
@@ -214,3 +212,24 @@ finalized_l1_contract_events.transaction_hash AS finalized_l1_transaction_hash`)
 
 	return withdrawals, nil
 }
+
+/*
+// LatestEpoch returns the latest epoch in which a deposit or withdrawal was observed
+func (db *bridgeTransfersDB) LatestEpoch() (*Epoch, error) {
+	var deposit L1BridgeDeposit
+	result := db.gorm.Table("l1_bridge_deposits").Order("l1_bridge_deposits.timestamp DESC").Take(&deposit)
+	if result.Error != nil {
+		if errors.Is(result.Error, gorm.ErrRecordNotFound) {
+			return nil, nil
+		}
+		return nil, result.Error
+	}
+
+	depositsQuery := db.gorm.Table("l1_bridge_deposits").Select("MAX(l1_bridge_deposits.timestamp) as latest_deposit_time")
+	depositsQuery = depositsQuery.Joins("l1_block_headers").Where("l1_block_headers.timestamp <= latest_deposit_time")
+	depositsQuery = depositsQuery.Joins("INNER JOIN l2_block_headers ON l1_block_headers.timestamp = l2_block_headers.timestamp")
+	depositsQuery = depositsQuery.Select("l1_block_headers.*")
+
+	return nil, nil
+}
+*/
```

Comment:
```suggestion
```
This got moved elsewhere it looks like

Reply:
Ah no this is intended, just didn't get around to finishing it.

1. `BridgeTransfers.LastestEpoch`: detects the last epoch indexed for BridgeTransfers as this is where the bridge processor should start from. This will allow the bridge processor to pick up where it left on from on when calling `NewBridgeProcessor`.

2. `Blocks.LastestEpoch` is the last epoch indexed between the L1 & L2 ETLs. This is used by the bridge processor to understand how much forward it progress relative to the L1/L2 ETL.

Will definitely be adding some comments to these functions before merging

Reply:
Since the E2E tests pass as it always starts from genesis. This is one of the TODOs i was leaving to quickly implement tomorrow

Reply:
Dont need this anymore